### ISSUE TITLE
transformer implementation has no residual

### ISSUE THREAD

**Issue by chengmengli06** [tensorflow/models#7533]
https://github.com/tensorflow/models/blob/master/official/transformer/model/transformer.py
The transfomer implementation seems to be incorrect. I cannot find the residual component, and the batchnorm is missed after every feed forward layer.

**Comment by chengmengli06** [comment]
I find it
